openc3/lib/openc3/microservices/interface_microservice.rb

```diff
@@ -428,10 +429,12 @@ def run
               params = JSON.parse(msg_hash['params'], allow_nan: true, create_additions: true)
             end
             @router = @tlm.attempting(*params)
+            next 'SUCCESS'
           end
           if msg_hash['disconnect']
             @logger.info "#{@router.name}: Disconnect requested"
             @tlm.disconnect(false)
+            next 'SUCCESS'
```

Comment:
Not sure how this worked without the `next 'SUCCESS'` but this matches the rest of the behavior